src/Sentry.Unity.Editor.iOS/BuildPostProcess.cs

```diff
@@ -160,6 +146,38 @@ internal static void SetupSentry(SentryUnityOptions options,
         logger.LogInfo("Successfully added Sentry to the Xcode project.");
     }
 
+    internal static void SetupSentryCli(SentryUnityOptions options,
+        SentryCliOptions? cliOptions,
+        IDiagnosticLogger logger,
+        string pathToProject)
+    {
+        if (cliOptions is null)
+        {
+            logger.LogWarning("No Sentry CLI options provided. You can create them by opening the Configuration Window at Tools -> Sentry");
+            return;
+        }
+
+        logger.LogInfo("Setting up Sentry CLI.");
+
+        if (options.Il2CppLineNumberSupportEnabled && cliOptions.IsValid(logger, EditorUserBuildSettings.development))
+        {
+            logger.LogWarning("The IL2CPP line number support requires the debug symbol upload to be enabled.");
+        }
```

Comment:
We want to log configuration that is allowed but that we already know might not yield the expected result. I.e. the user has `LineNumberSupport` enabled but skips the symbol upload.